tfrecorder/client_test.py

```diff
@@ -224,12 +189,14 @@ def setUp(self):
     self.header = data.pop(0)
     self.sample_data = data
 
-  def test_valid_csv_no_header_no_names_specified(self):
-    """Tests a valid CSV without a header and no header names given."""
-    f = _make_csv_tempfile(self.sample_data)
-    actual = client.read_csv(f.name, header=None)
-    self.assertEqual(list(actual.columns), constants.IMAGE_CSV_COLUMNS)
-    self.assertEqual(actual.values.tolist(), self.sample_data)
+  # TODO(mikebernico) Talk with Carlos and depricate.
```

Comment:
Can't this default to the 'Image CSV' schema?

Reply:
yeah, that makes sense. I updated it to default to image csv.